Dockerfile

```
@@ -12,4 +12,7 @@ EXPOSE ${PORT}
 
 COPY .docker/run.sh ./
 
+COPY tests/test_config.json ./
```

I would make this path part of `docker-compose.yml` as an argument, which then defaults to `tests/test_config.json` in the `Dockerfile`.
Then my suggested added comment should also be shifted to to the `ARG` line.
```suggestion
COPY tests/test_config.json ./  # Should be updated with your config JSON file
```